org-cyf-guides/content/contributing/minutes/index.md

```diff
@@ -17,6 +17,97 @@ Attendees:
 
 ---
 
+## 2024-12-03
+
+Attendees: Ali Smith, Sally McGrath, Mitch Lloyd
+
+### ❗Actions from last time
+
+* [ ] All: Collaboratively put together LOs and name for ITP Introduction Module
+* [ ] Daniel: Sync with Karen on Piscine dashboard requirements
```

Comment:
```suggestion
* [x] Daniel: Sync with Karen on Piscine dashboard requirements
```

We did this! tl;dr: Short-term we're going to run the application process outside of the dashboard, and will use the dashboard to track progress _within_ the Piscine. Long-term they know what we need for application stuff too.

Reply:
Thanks! I had a feeling you had, I remember seeing slack activity about it, but didn't want to complete without double checking.